chrysom/README.md

```diff
@@ -0,0 +1,36 @@
+# chrysom
+The client library for communicating with Argus over HTTP.
+[![PkgGoDev](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)](https://pkg.go.dev/github.com/xmidt-org/argus@v0.5.1/chrysom)
+
+## Summary
+This package enables CRUD operations with Argus: 
```

Comment:
Can you add a bit more to this? It should be like a one to two sentence summary of chrysom that would make sense even if it was copied and pasted outside of this readme.